Create +Page.mdx

thednp opened the pull request:
Initial VanJS commit.

brillout commented:
I wouldn't mention `vike-vanjs` since it doesn't exist.

thednp commented:
My bad, will come back to it tomorrow morning, I know what to do for that. Please instruct on other issues after that.

brillout commented:
>  I need to create some simple starter templates

No need: we can just remove the `vike-vanjs` mention and merge the PR as-is. But, if you want, adding more examples could be neat indeed.

Adding VanJS to [Bati](https://github.com/vikejs/bati) would take quite some time, it's far from trivial. It would probably be easier to fork Bati.

thednp commented:
@brillout

I've added 2 examples to my [patch-1/examples](https://github.com/thednp/vike/tree/patch-1/examples)  folder but they don't show in Github. Very strange. Any idea?

Also I've updated `/van/+Page.mdx` and `/ui-frameworks`, in the same branch.

thednp commented:
@brillout I think this PR can be closed. It doesn't include examples.
The other PR #2281 should be complete.

Notes:
- I forked the react example and adapted it for 2 VanJS examples: simple and JSX
- I tried to update the tests, you might want to look into that
- let me know if any issue.
